### ISSUE TITLE
Create `workbench.editor.openPositioning` pendant for tab groups

### ISSUE THREAD

**Issue by EnCey** [microsoft/vscode#56142]
# Steps to reproduce
1. Open a markdown file via explorer (`ctrl+shift+e`) and then its preview (`ctrl+k, v`)
2. Open a second markdown file via explorer
  **Result:** markdown file opened in a new tab in the left tab group, its preview replaced the previous one in the right tab group
3. Open a third markdown file via explorer
  **Result:** markdown file opens **in the 2nd tab group** next to the preview and **spawns a 3rd tab group**

# Expected behavior
Markdown files should always be opened in the left tab group and their preview in the 2nd tab group.
This behavior is independent of other tab settings (open markdown file as preview, do not use tabs at all, etc.)

# The problem
It seems that after step 2) the preview tab in the 2nd tab group has focus. This is why in step 3), the markdown file is opened in the 2nd tab group.

This process can be repeated indefinitely. After step 3), the preview in tab group 3 has focus. If another markdown file is opened, it will open in the 3rd tab group and spawn a 4th group.

# Why this matters
When editing markdown files in a folder, you often have to switch between files.
I cannot fathom a scenario where someone would want a new markdown file to be opened next to a preview tab, effectively hiding the preview and displaying the 2 markdown files side-by-side, plus showing a preview of the 2nd in a 3rd tab group.

# Possible solution
Create a setting similar to `workbench.editor.openPositioning`, only that it controls which **tab group** is used when opening new files. I.e. when setting this to `first` it would solve the problem described here.

**Comment by EnCey** [comment]
This is perhaps related to, but not a duplicate of #19339.
The issue here is simply that you cannot quickly open a bunch of markdown files via the explorer view without ending up with 3+ tab groups.

**Comment by mjbvz** [comment]
Closing due to lack of further feedback since this was opened 2 years ago
